.env.example

```diff
@@ -6,6 +6,7 @@ COMPOSE_PATH_SEPARATOR=;
 COMPOSE_FILE=docker-compose.yml;docker/dev.yml
 
 API_PORT=127.0.0.1:8000
+OAUTH2_SERVER_URL=https://world.openfoodfacts.org/cgi/auth.pl
```

Comment:
We will deploy it in staging as well (https://world.openfoodfacts.net), so the URL need not to be hardcoded. You can use openfoodfacts SDK URLBuilder (https://github.com/openfoodfacts/openfoodfacts-python/blob/develop/openfoodfacts/utils.py#L23) to build OFF URLs this easily.

Reply:
if it's in the .env then it's not hardcoded, right ? :sweat_smile: 
I'm not sure how to do this so I'll do it in a seperate PR :)